### ISSUE TITLE
Question about the evaluation results

### ISSUE THREAD

**Issue by thuxyz19** [TIGER-AI-Lab/VLM2Vec#20]
Hi~The proposed multimodal embedding model built upon MLLM is amazing! I have a question about the evaluation results shown here. Is it produced by VLM2Vec-Phi3.5V or the stronger model VLM2Vec-LLaVa-Next? If it is produced by VLM2Vec-Phi3.5V, can you also share the evaluation results of VLM2Vec-LLaVa-Next? I wonder how much improvement it can make when changing the model size from 4B to 7B. 
<img width="823" alt="image" src="https://github.com/user-attachments/assets/cb9a5053-73e2-4ca6-b8e7-4f228e2ff0a9" />

**Comment by XMHZZ2018** [comment]
Hi @thuxyz19 ,

Thank you for your interest in our work! I have updated the experimental results figure in README to include the performance of VLM2Vec-LlaVa-Next on MMEB eval. Compared to the Phi-3.5-v version, VLM2Vec-LlaVa-Next shows clear improvements in classification, retrieval, and visual grounding tasks. We also evaluated the model on additional datasets, such as Flickr30K, and observed significant improvements there as well. We will update all these results on arXiv soon.
